swagger.json

```diff
@@ -70,7 +70,7 @@
     },
     "/v1/watch-graphs": {
       "put": {
-        "operationId": "GraphController_watchGraphs",
+        "operationId": "GraphControllerV1_watchGraphs",
```

Comment:
I thought this affected the openApi index.html headers. May be wrong.

Reply:
It doesn't now because of the tags... Or at least that's what I thought